<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MojeDżwięki, Wykonał: 05251202599</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #2E8B57
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="previous_button">
    <property name="geometry">
        <rect>
            <x>20</x>
            <y>120</y>
            <width>10</width>
            <height>70</height>
        </rect>
    </property>
    <property name="text">
        <string/>
    </property>
    <property name="styleSheet">
        <string notr="true">
                background-image: url('obraz3.png')
        </string>
    </property>
</widget>
<widget class="QPushButton" name="next_button">
    <property name="geometry">
        <rect>
            <x>840</x>
            <y>120</y>
            <width>10</width>
            <height>70</height>
        </rect>
    </property>
    <property name="text">
        <string/>
    </property>
    <property name="styleSheet">
        <string notr="true">
                background-image: url('obraz2.png')
        </string>
    </property>
</widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>obraz.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="artist_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Wykonawca</string>
    </property>
   </widget>
   <widget class="QLabel" name="album_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white
</string>
    </property>
    <property name="text">
     <string>Album</string>
    </property>
   </widget>
   <widget class="QLabel" name="songs_number_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #61D918
</string>
    </property>
    <property name="text">
     <string>Ilość utworów</string>
    </property>
   </widget>
   <widget class="QLabel" name="year_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #61D918</string>
    </property>
    <property name="text">
     <string>Rok</string>
    </property>
   </widget>
   <widget class="QLabel" name="download_number_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #61D918</string>
    </property>
    <property name="text">
     <string>Ilość odtworzeń</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #61D918;
color: black;</string>
    </property>
    <property name="text">
     <string>Pobierz</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
